tracks/go/exercises/pig-latin/mentoring.md

```diff
@@ -0,0 +1,72 @@
+# Pig Latin
+
+### Reasonable solutions:
+
+There is a (somewhat) reasonable solution using regular expressions, however most students are likely to use a combination of `strings` funtions or string slicing to solve this problem.
```

Comment:
This section has a bunch of components but not a complete solution. You may want to present a complete solution here. If you want to explain it, you can use functions and comments and delve into details in following sections.

Reply:
The three parts/functions together form a complete solution. I can try to reorganize if you think that would help it flow better.